tests/ApiBasedPageRetrieverTest.php

```diff
@@ -156,4 +156,34 @@ public function testWhenConstructingWithInvalidFetchMode_exceptionIsThrown() {
 		);
 	}
 
+	public function testRetrieverReturnsApiResultInRawExpandedMode() {
+		$pageName = 'Raw_text';
+
+		$this->api->method( 'isLoggedin' )->willReturn( true );
+		$this->api->method( 'postRequest' )
+			->with( new SimpleRequest( 'query', $this->newParamsForExpandedTemplates( $pageName ) ) )
+			->willReturn( $this->getJsonTestData( 'mwApiRawExpanded.json' ) );
+
+		$expectedContent = 'No curly brackets here.';
+
+		$this->pageRetriever = new ApiBasedPageRetriever(
+			$this->api,
+			$this->apiUser,
+			$this->logger,
+			self::PAGE_PREFIX,
+			ApiBasedPageRetriever::MODE_RAW_EXPANDED
+		);
+
+		$this->assertSame( $expectedContent, $this->pageRetriever->fetchPage( $pageName ) );
```

Comment:
I'd inline $expectedContent, now there is a block of unrelated code between where it is defined and where it is used